examples/elixir/IntegerTest.ex

```diff
@@ -0,0 +1,25 @@
+defmodule IntegerTest do
+  @compile {:no_warn_undefined, [Console]}
+
+  def start() do
```

Comment:
examples is no the right place for tests, but rather for simple demos or examples.